app/models/project.rb

```diff
@@ -30,6 +30,20 @@ def formatted_dollar_amount
     number_to_currency(price / 100.00)
   end
 
+  def downcase_categories
+    categories.map do |category|
+      category.name.downcase
+    end
+  end
+
+  # move this logic
+  def as_json(s)
```

Comment:
Unused method argument - `s`. If it's necessary, use `_` or `_s` as an argument name to indicate that it won't be used. You can also write as `as_json(*)` if you want the method to accept any arguments but don't care about them.